map-the-debris/map-the-debris.js

```diff
@@ -0,0 +1,21 @@
+function getOrbitalPeriod(avgAlt) {
+  var GM = 398600.4418;
```

Comment:
why are you oscillating between const, var and let? https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75